app/Geocoding/GeocodingServiceFactory.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Geocoding;
+
+use Geocoder\Provider\GoogleMaps\GoogleMaps;
+use Geocoder\StatefulGeocoder;
+use Http\Adapter\Guzzle7\Client;
+use Psr\Log\LoggerInterface;
+
+class GeocodingServiceFactory
+{
+    private bool $addLocationNameToCoordinatesLookup;
+
+    public function __construct(bool $isNewFeature)
```

Comment:
Maybe make the name of this variable more explicit and also use `$addLocationNameToCoordinatesLookup`